components/Storages.js

```diff
@@ -0,0 +1,38 @@
+import { nanoid } from 'nanoid';
+import Link from 'next/link';
+
+export default function Storages({ storages, setStorages }) {
+  function handleSubmitAdd(event) {
+    event.preventDefault();
+    const form = event.target.elements;
+    const newStorage = {
+      name: form.storage.value,
+      id: nanoid(),
+    };
+    setStorages([{ name, ...newStorage }, ...storages]);
+
```

Comment:
I'm not sure, what this is supposed to do - `newStorage` already contains a property `name`. Couldn't you just have `setStorages([newStorage, ...storages]);`?

Reply:
Hey Neele, I checked this function and this is the result:

```
[{newStorage: {name: 'test', id: 'kxiNewlVLlhYguHc_UxML'}}, 
{id: 345, name: "fridge"}]

``` 

You get an object inside and object. I think that's not want you want. I suggest to update it as Ralf suggested.

Reply:
Maybe this is also the reason why a new entry isn't rendered in the list below